dev/docs/assets/overrides.css

```diff
@@ -1,37 +1,11 @@
 /* Overrides for dartdoc styles. */
-body {
-  font-size: 15px;
-  font-family: Roboto, sans-serif;
-  line-height: 1.5;
-  color: #111;
-  background-color: #fdfdfd;
-  -webkit-font-smoothing: auto;
-}
-
-header {
-  background-color: white;
-  color: #424242;
-}
-
-nav.navbar {
-  min-height: 57px;
-  padding: 6px 0;
-}
 
-header.header-fixed nav.navbar-fixed-top {
-  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
-}
-
-h1 {
-  font-size: 42px !important;
-  letter-spacing: -1px;
-}
-
-h2 {
-  color: #111;
-  font-size: 24px;
+/* Prefer Google Sans to bring the API docs closer to the main site's look */
+body {
+  font-family: "Google Sans", "Roboto", sans-serif;
```

Comment:
`Google Sans` is intended only for headers, not body text.

Reply:
Ahh, good point.  OK, I revamped it to only apply to headers.